### ISSUE TITLE
[customize-titlebar] conditional labels

### ISSUE THREAD

**Issue by roytam1** [mozilla/nightlytt#113]
I think it will be nice to have a foobar2000 $if() label so that people can show page URL if page title is blank, or not showing separator if page title is blank.

for now I can just hack browser.js and change tabTitle() to:

``` javascript
  return tabbrowser.mCurrentBrowser.contentTitle ? tabbrowser.mCurrentBrowser.contentTitle : tabbrowser.mCurrentBrowser.contentDocument.location.href;
```

**Comment by whimboo** [comment]
Yes it is. Due to time constraints nightlytt is basically unmaintained at the moment. But we are happy to accept patches in case someone is interested to help out.
